wg-data-protection/charter.md

```diff
@@ -0,0 +1,76 @@
+# WG Data Protection Charter
+
+This charter adheres to the [wg-governance] guidance as well as
+the general conventions described in the [Kubernetes Charter README] and
+the Roles and Organization Management outlined in [sig-governance], where
+applicable to a Working Group.
+
+
+## Scope
+
+Data protection in Kubernetes context typically involves backup and recovery
+of two types of entities:
+* Kubernetes object resources
+* Persistent volume data
+We consider it a complicated and layered problem, including backup and recovery
+at persistent volume level, application level, and cluster level. Part of the
```

Comment:
what does `application` mean? Is it a kubernetes object like - a StatefulSet, Deployment, or does it include a collection of objects (deployments, statefulsets and daemonsets) that together form an application? 

Probably, `complicated and layered problem` means just that - it can be volume, application or cluster and also combination of multiple applications.

Reply:
I believe this is related to https://github.com/kubernetes/enhancements/pull/1051, so a collection of objects that form an application.

Reply:
@kmova that's a really interesting point.
as @prydonius commented, while there is an ongoing KEP from @liyinan926 focusing on statefulset. there also have been very initial discussions around how to backup/restore a set of objects which form an application consistently (i.e., a wordpress application).
Again, those are exactly the topics the proposed WG try to collect thoughts, organize discussions, and come up with a definition.

Reply:
Just to clarify, the KEP https://github.com/kubernetes/enhancements/pull/1051 focuses on the APIs for defining and performing data protection operations on stateful applications, which consist of a collection of related Kubernetes resources (including PVCs) that make up the applications.